src/codecs/jats/__file_snapshots__/06908fc3-73df-1014-bb56-a21daa237ef0.json

```diff
@@ -1595,6 +1595,10 @@
         "type": "Date",
         "value": "2015"
       },
+      "isPartOf": {
+        "type": "Periodical",
+        "name": "Nature"
+      },
```

Comment:
@fred-atherden I am happy with this adjustment with the logic. It would be great for you to scan over all/some of the changes to the fixture in the __file_snapshots__ folder as they represent the alteration to the output of the conversion because of the adjustments we have made to the logic.

I have removed the addition of `meta.type` to the decoded reference so we can reduce the differences in this review. As soon as this is merged in I will prepare that PR as a followup.

Reply:
Thanks for this @nlisgo, I've taken a look at the changes to the file snapshots and they're looking good. We're surfacing more information as desired. I think we're good to go ahead with this - thanks for your help!